src/app/src/util/constants.oarmap.js

```diff
@@ -0,0 +1,124 @@
+export const MAPBOX_TOKEN = process
+    .env
+    .REACT_APP_MAPBOX_TOKEN;
+
+export const CIRCLE_COLOR = process
+    .env
+    .REACT_APP_MAP_COLOR;
+
+export const CIRCLE_TEXT_COLOR = '#fff';
+
+export const CIRCLE_COLOR_SHADOW = process
+    .env
+    .REACT_APP_MAP_COLOR_TRANSPARENT;
+
+export const FACILITIES_SOURCE = 'FACILITIES_SOURCE';
+
+export const initialViewport = Object.freeze({
+    lat: 34,
+    lng: 5,
+    zoom: 1.5,
+});
+
+export const CLUSTER_MAX_ZOOM = 100;
+export const CLUSTER_RADIUS = 25;
+
+export const oarMapLayerIDEnum = Object.freeze({
+    points: 'points',
+    pointsShadow: 'pointsShadow',
+    clusterCount: 'clusterCount',
+    unclusteredPoint: 'unclusteredPoint',
+    highlightedPoint: 'highlightedPoint',
+});
+
+export const oarMapLayers = Object.freeze([
```

Comment:
Yeah, this was extracted from this `addMapLayers` method in the legacy Map.jsx component:

https://github.com/open-apparel-registry/open-apparel-registry/blob/develop/src/app/src/components/Map.jsx#L386

Aside from setting some values to constants, I didn't otherwise adjust it.